lib/web/sessions.go

```diff
@@ -623,6 +623,8 @@ type sessionCacheOptions struct {
 	sessionLingeringThreshold time.Duration
 	// proxySigner is used to sign PROXY header and securely propagate client's real IP
 	proxySigner multiplexer.PROXYHeaderSigner
+	// See [sessionCache.startWebSessionWatcherImmediately]. Used for testing.
+	startWebSessionWatcherImmediately bool
```

Comment:
This would always be set to zero currently. It will also be ambiguous if it's unset, which will push us to `*time.Duration` or some other shenanigans.